kolibri/utils/options.py

```diff
@@ -304,135 +257,207 @@ def origin_or_port(value):
-            "envvars": (
-                "KOLIBRI_CENTRAL_CONTENT_BASE_URL",
-                "CENTRAL_CONTENT_DOWNLOAD_BASE_URL",
-            ),
+            "envvars": ("CENTRAL_CONTENT_DOWNLOAD_BASE_URL",),
+            "description": """
+                URL to use as the default source for content import.
+                Slightly counterintuitively this will still be displayed in the UI as 'import from Kolibri Studio'.
+            """,
         },
         "DATA_PORTAL_SYNCING_BASE_URL": {
             "type": "string",
             "default": "https://kolibridataportal.learningequality.org",
-            "envvars": ("KOLIBRI_DATA_PORTAL_SYNCING_BASE_URL",),
+            "description": "URL to use as the target for data portal syncing.",
         },
     },
     "Deployment": {
         "HTTP_PORT": {
             "type": "port",
             "default": 8080,
-            "envvars": ("KOLIBRI_HTTP_PORT", "KOLIBRI_LISTEN_PORT"),
+            "envvars": (
+                "KOLIBRI_HTTP_PORT",
+                "KOLIBRI_LISTEN_PORT",
+            ),
+            "description": "Sets the port that Kolibri will serve on. This can be further overridden by command line arguments.",
+        },
+        "RUN_MODE": {
+            "type": "string",
+            "description": "Used to flag non-user Kolibri instances",
+            "skip_blank": True,
         },
-        "RUN_MODE": {"type": "string", "envvars": ("KOLIBRI_RUN_MODE",)},
         "DISABLE_PING": {
             "type": "boolean",
             "default": False,
-            "envvars": ("KOLIBRI_DISABLE_PING",),
+            "description": "Turn off the statistics pingback. This will also disable update notifications",
         },
         "URL_PATH_PREFIX": {
-            "type": "string",
+            "type": "url_prefix",
             "default": "/",
-            "envvars": ("KOLIBRI_URL_PATH_PREFIX",),
-            "clean": lambda x: x.lstrip("/").rstrip("/") + "/",
+            "description": """
+                Serve Kolibri from a subpath under the main domain. Used when serving multiple applications from
+                the same origin. This option is not heavily tested, but is provided for user convenience.
+            """,
         },
         "LANGUAGES": {
             "type": "language_list",
             "default": SUPPORTED_LANGUAGES,
-            "envvars": ("KOLIBRI_LANGUAGES",),
+            "description": """
+                The languages that this Kolibri should enable. The default is all the languages that Kolibri supports.
+                Can either be a single language code, or a comma separated list of language codes.
+            """,
         },
         "ZIP_CONTENT_ORIGIN": {
             "type": "origin_or_port",
             "default": "",
-            "envvars": ("KOLIBRI_ZIP_CONTENT_ORIGIN",),
+            "description": """
+                When running in default operation, this will default to blank, and the Kolibri
```

Comment:
When running by default (value blank), Kolibri frontend looks for the zipcontent endpoints on the same domain as Kolibri proper, but uses ZIP_CONTENT_PORT instead of HTTP_PORT.